jobs/loc/TranslationsImportExport.yml

```diff
@@ -0,0 +1,56 @@
+# ==================================================================================
+# Pipeline for VsCodeExtension-Localization build definition
+# Runs OneLocBuild task to localize xlf file
+# ==================================================================================
+
+resources:
+  repositories:
+  - repository: self
+    clean: true
+
+trigger: none
+pr: none
+schedules:
+- cron: "0 7 * * *"
+  displayName: Daily 7 AM
+  branches:
+    include:
+    - main
+    - localization
+  always: true
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- task: CmdLine@2
+  inputs:
+    script: 'yarn install'
+
+- task: CmdLine@2
+  inputs:
+    script: 'yarn run translations-export'
+
+- task: OneLocBuild@2
+  env:
+    SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+  inputs:
+    locProj: 'jobs/loc/LocProject.json'
+    outDir: '$(Build.ArtifactStagingDirectory)'
+    isCreatePrSelected: false
+    prSourceBranchPrefix: 'locfiles'
+    packageSourceAuth: 'patAuth'
+    patVariable: '$(OneLocBuildPat)'
+    LclSource: lclFilesfromPackage
+    LclPackageId: 'LCL-JUNO-PROD-VMAKEFILE'
```

Comment:
Did you get this ID from the loc team?  If not, you'll want to get an ID from them.

Reply:
I just updated after copy paste. Thank you for explaining what needs to happen and I'll contact the loc team once I get their contact from you.